File: Lógica de Programação Java e C#/4 - Controle de Fluxo/C#/Exercícios Complementares/ComparaValores.cs
class ComparaValores
{
    static void Main()
    {
        System.Random gerador = new System.Random();
        double valor1 = gerador.NextDouble();
        double valor2 = gerador.NextDouble();
        System.Console.WriteLine(valor1);
        System.Console.WriteLine(valor2);
        if (valor1 > valor2)
        {
            System.Console.WriteLine("Primeiro > Segundo");
        }
        if (valor1 < valor2)
        {
            System.Console.WriteLine("Segundo > Primeiro");
        }
        else if (valor1 == valor2)
        {
            System.Console.WriteLine("Primeiro== Segundo");
        }
           
        }
}

File: Lógica de Programação Java e C#/4 - Controle de Fluxo/Java/Exercícios Complementares/ComparaValores.java
class ComparaValores{
	public static void main(String [] args){
		double valor1 = Math.random();
			System.out.println(valor1);
		double valor2 = Math.random();
			System.out.println(valor2);	
	if(valor1 < valor2)
		System.out.println("Segundo > Primeiro");
	if(valor1 > valor2)
		System.out.println("Primeiro > Segundo");
	if(valor1 == valor2)
		System.out.println("Primeiro = Segundo");
		}
}